RELEASE.md

```diff
@@ -83,6 +86,8 @@ For a **PATCH** release:
   - Add a new changelog for the release
   - Update the cargo.toml versions for the bpfd-api, bpfd, and bpfctl crates
   - Update the bpfd-operator version in it's MAKEFILE and run `make bundle` to update the bundle version
+    this will generate a new `/bpfd-operator/bundle` directory which will ONLY be tracked in the
```

Comment:
```suggestion
    This will generate a new `/bpfd-operator/bundle` directory which will ONLY be tracked in the
```
And previous line needs a `.`
Line 84 also needs a `.`

Reply:
Done thanks!